frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -199,7 +199,7 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
   const getLastMessageWithSuggestedReplies = useCallback(() => {
     const lastMessages = props.messages
       ?.filter((message: Message) => message.senderType == 'source_contact')
-      .slice(props.messages.length - 5)
+      .slice(-5)
```

Comment:
This was a stupid mistake 🤦

Reply:
This is a separate point, but if I re-read the ticket correctly this code also doesn't quite do what is asked.

The difference is subtle, but the original ticket read to me like they want the suggestions to be the union of all the suggestions of the previous 5 messages. Not the first one that has any.